lib/scanner/index.js

```diff
@@ -30,13 +40,22 @@ class Scanner {
   }
 
   /**
-   * Start scanning.
+   * Perform a scan.
    *
    * @public
+   * @param  {Number} duration
+   * @return {Promise}
    */
-  start() {
-    debug('start');
-    this.cycle();
+  scan(duration) {
```

Comment:
Start suggests we have to call stop. This higher-level api allows the user to scan for a given time period, which is largely what apps want to do.